cdrs-tokio/Cargo.toml

```diff
@@ -37,6 +37,7 @@ tokio = { version = "1", features = ["net", "io-util", "rt", "sync", "macros", "
 tokio-rustls = { version = "0.22", optional = true }
 uuid = "0.8"
 webpki = { version = "0.21", optional = true }
+cdrs-tokio-helpers-derive = { version = "3.0", path = "../cdrs-tokio-helpers-derive" }
```

Comment:
Was it necessary to move from dev-depndencies?

Reply:
I was using the macros to parse the system peers table.